modules/ROOT/pages/monitor-api-conformance.adoc

```diff
@@ -66,8 +81,9 @@ On the main API Governance console page, select the *APIs* tab and then click th
 * Number of nonconformant APIs (fail)
 * Timestamps
 
-=== Specific Profile Report
-On a selected governance profile page, click the more options menu icon (image:overflow-menu-icon.png[width=9,height=15,fit=line,title="Three vertical dots"]) and then select *Export Reports* to export a report that shows the selected governance profile's information, such as: 
+[[gov-spec-profile-export]]
+=== Specific Profile Export
+On a selected governance profile page, click the more options menu icon (image:overflow-menu-icon.png[width=9,height=15,fit=line,title="Three vertical dots"]) and then select *Export to CSV* to export the selected governance profile's information, such as: 
```

Comment:
Is title the same as alt text? Is it necessary?

Reply:
This is the "kebab", so I felt it might be useful to describe the visual in the alt text. Thinking about it, though, I guess the alt text is for people who can't see the visual anyway, and they just need to know the functional use of it. I'll remove the alt text for these throughout as time allows.